tests/callbacks/test_lr_monitor.py

```diff
@@ -1,109 +1,135 @@
+def test_lr_monitor_multi_lrs(tmpdir, logging_interval):
     """ Test that learning rates are extracted and logged for multi lr schedulers. """
     tutils.reset_seed()
 
     model = EvalModelTemplate()
     model.configure_optimizers = model.configure_optimizers__multiple_schedulers
 
-    lr_logger = LearningRateLogger(logging_interval=logging_interval)
+    lr_monitor = LearningRateMonitor(logging_interval=logging_interval)
     trainer = Trainer(
         default_root_dir=tmpdir,
         max_epochs=2,
         limit_val_batches=0.1,
         limit_train_batches=0.5,
-        callbacks=[lr_logger],
+        callbacks=[lr_monitor],
     )
     result = trainer.fit(model)
     assert result
 
-    assert lr_logger.lrs, 'No learning rates logged'
-    assert len(lr_logger.lrs) == len(trainer.lr_schedulers), \
+    assert lr_monitor.lrs, 'No learning rates logged'
+    assert len(lr_monitor.lrs) == len(trainer.lr_schedulers), \
         'Number of learning rates logged does not match number of lr schedulers'
-    assert all([k in ['lr-Adam', 'lr-Adam-1'] for k in lr_logger.lrs.keys()]), \
+    assert all([k in ['lr-Adam', 'lr-Adam-1'] for k in lr_monitor.lrs.keys()]), \
         'Names of learning rates not set correctly'
 
     if logging_interval == 'step':
         expected_number_logged = trainer.global_step
     if logging_interval == 'epoch':
         expected_number_logged = trainer.max_epochs
 
-    assert all(len(lr) == expected_number_logged for lr in lr_logger.lrs.values()), \
+    assert all(len(lr) == expected_number_logged for lr in lr_monitor.lrs.values()), \
         'Length of logged learning rates do not match the expected number'
 
 
-def test_lr_logger_param_groups(tmpdir):
+def test_lr_monitor_param_groups(tmpdir):
     """ Test that learning rates are extracted and logged for single lr scheduler. """
     tutils.reset_seed()
 
     model = EvalModelTemplate()
     model.configure_optimizers = model.configure_optimizers__param_groups
 
-    lr_logger = LearningRateLogger()
+    lr_monitor = LearningRateMonitor()
     trainer = Trainer(
         default_root_dir=tmpdir,
         max_epochs=2,
         limit_val_batches=0.1,
         limit_train_batches=0.5,
-        callbacks=[lr_logger],
+        callbacks=[lr_monitor],
     )
     result = trainer.fit(model)
     assert result
 
-    assert lr_logger.lrs, 'No learning rates logged'
-    assert len(lr_logger.lrs) == 2 * len(trainer.lr_schedulers), \
+    assert lr_monitor.lrs, 'No learning rates logged'
+    assert len(lr_monitor.lrs) == 2 * len(trainer.lr_schedulers), \
         'Number of learning rates logged does not match number of param groups'
-    assert all([k in ['lr-Adam/pg1', 'lr-Adam/pg2'] for k in lr_logger.lrs.keys()]), \
+    assert all([k in ['lr-Adam/pg1', 'lr-Adam/pg2'] for k in lr_monitor.lrs.keys()]), \
         'Names of learning rates not set correctly'
+
+
+def test_lr_logger_deprecated(tmpdir):
```

Comment:
pls, move this to file with testing deprecated staff... later much easier to find